src/cloud-api-adaptor/pkg/agent/update_test.go

```diff
@@ -1,227 +0,0 @@
-package agent
```

Comment:
Would any of the unit tests from here be useful to test agent/config.go - not to do the update but to check the config file is created as expected?